src/query-builder/SelectQueryBuilder.ts

```diff
@@ -2877,6 +2877,11 @@ export class SelectQueryBuilder<Entity extends ObjectLiteral>
                     })
                 })
             } else {
+                if (column.isVirtualProperty) {
```

Comment:
@tolgap This just removes the VirtualColumn feature instead of fixing the issue that it has.

Probably this should've been solved in `RawSqlResultsToEntityTransformer`.